src/Controllers/CheckoutController.php

```diff
@@ -129,6 +295,46 @@ public function doKlarnaAuth(
         return $this->getJsonSuccess($auth);
     }
 
+    /**
+     * @param PaymentService $paymentService
+     * @param  $order
+     * @return string
+     */
+    public function doKlarnaAuthForReinit(
+        $orderId,
+        PaymentService $paymentService
+    ) {
```

Comment:
Should the loaded order or the ID be entered here?
I think the documentation is not correct at this point.